sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -35,6 +35,7 @@ def analyze_custom_documents(custom_model_id):
         os.path.join(os.path.abspath(__file__), "..", "./sample_forms/forms/Form_1.jpg")
     )
     # [START analyze_custom_documents]
+    from prettytable import PrettyTable
```

Comment:
I'm not sure if this is a good idea to introduce a third-party library. I didn't see other projects using it.

Reply:
fixed , thx